charts/consul/templates/crd-proxydefaults.yaml

```diff
@@ -204,6 +195,16 @@ spec:
                   your services secure, we recommend using "strict" mode whenever
                   possible and enabling "permissive" mode only when necessary.'
                 type: string
+              prioritizeByLocality:
+                description: PrioritizeByLocality controls whether the locality of
+                  services within the local partition will be used to prioritize connectivity.
+                properties:
+                  mode:
+                    description: 'Mode specifies the type of prioritization that will
+                      be performed when selecting nodes in the local partition. Valid
+                      values are: "" (default "none"), "none", and "failover".'
+                    type: string
+                type: object
```

Comment:
Just wanted to add that this should be fine; I didn’t understand how to properly generate changes to this CRD in my first PR updating it and was going to circle back to fix this.